src/main/java/com/treasuredata/client/TDRequestErrorHandler.java

```diff
@@ -258,6 +258,11 @@ private static String parseConflictsWith(TDApiErrorMessage errorResponse)
             try {
                 content = Optional.of(response.body().string());
             }
+            catch (SocketTimeoutException e) {
```

Comment:
Is this the exception that was actually happening? How about other regular TimeoutException, ExecutionException, etc?

Reply:
They're not IOException, so it would be caught other places and messages would be different.